<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="menuPage">
     <widget class="QPushButton" name="encryptPageButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>271</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decryptPageButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>160</y>
        <width>271</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="encryptPage">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="pswdLineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>410</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mot de passe:</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="pswdBtnBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>440</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="imageLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>10</y>
        <width>761</width>
        <height>371</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="decryptPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>271</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>49</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Page2</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="menuPageButton">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>480</y>
      <width>781</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imageLabel</class>
   <extends>QLabel</extends>
   <header location="global">custom.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressources/ressource1.qrc"/>
 </resources>
 <connections/>
</ui>
